src/org/labkey/test/util/ExtHelper.java

```diff
@@ -565,8 +565,9 @@ private static String jsString(String s)
          */
         public static Locator.XPathLocator extDialog(String title)
         {
-            return Locator.xpath("//span[" + Locator.NOT_HIDDEN + " and contains(@class, 'window-header-text') and contains(string(), '" + title + "')]");
-        }
+            return Locator.xpath("//div").withClass("x4-window").notHidden().
+                    withDescendant(Locator.xpath("//span").withClass("x4-window-header-text").withText(title));
+     }
```

Comment:
This Locator isn't intended to find the window, just the title bar. It also needs to work with ext3 and we shouldn't impose a strict text match where there wasn't one before.
```suggestion
            return Locator.tag("span").notHidden().withAttributeContaining("class", "window-header-text").containing(title);
         }
```